script/cart.js

```diff
@@ -0,0 +1,147 @@
+"strict";
+const cartList = document.querySelector(".cart-table-content");
+const cartItems = document.querySelector(".cart-items");
+const cartTotal = document.querySelector(".cart-total");
+const clearCartBtn = document.querySelector(".cart-clear");
+
+let cart = [];
+let cartItemEvent = [];
+
+//////////////Model//////////////////
+
+class View {
+  initialApp() {
+    cart = Storage.getCart();
+    this.setcartValues();
+  }
+
+  setcartValues() {
+    let tempTotal = 0;
+    let itemsTotal = 0;
+    cart.map((item) => {
+      tempTotal += item.price * item.amount;
+      itemsTotal += item.amount;
+    });
+    cartItems.innerText = itemsTotal;
+    cartTotal.innerText = tempTotal;
+  }
+
+  removeItem(id) {
+    cart = cart.filter((item) => item.id !== id);
+    this.setcartValues(cart);
```

Comment:
```suggestion
    this.setcartValues();
```
truyền này có ý nghĩa gì hơm

Reply:
hem có nghĩa luôn :v